pallets/lbp/src/benchmarking.rs

```diff
@@ -139,19 +139,94 @@ benchmarks! {
 		let max_limit: Balance = 1_000_000_000;
 		let pool_id = LBP::<T>::pair_account_from_assets(ASSET_A_ID, ASSET_B_ID);
 
-		LBP::<T>::create_pool(RawOrigin::Root.into(), caller.clone(), ASSET_A_ID, ASSET_A_AMOUNT, ASSET_B_ID, ASSET_B_AMOUNT, INITIAL_WEIGHT, FINAL_WEIGHT, WeightCurveType::Linear, DEFAULT_FEE, fee_collector.clone(), 0)?;
+		LBP::<T>::create_pool(RawOrigin::Root.into(), caller.clone(), ASSET_A_ID, ASSET_A_AMOUNT, ASSET_B_ID, ASSET_B_AMOUNT, INITIAL_WEIGHT, FINAL_WEIGHT, WeightCurveType::Linear, DEFAULT_FEE, fee_collector, 0)?;
 		ensure!(PoolData::<T>::contains_key(&pool_id), "Pool does not exist.");
 
 		let start = T::BlockNumber::from(1u32);
 		let end = T::BlockNumber::from(11u32);
 
 		LBP::<T>::update_pool_data(RawOrigin::Signed(caller.clone()).into(), pool_id, None, Some(start), Some(end), None, None, None, None, None)?;
 
+		frame_system::Pallet::<T>::set_block_number(T::BlockNumber::from(2u32));
+
 	}: _(RawOrigin::Signed(caller.clone()), asset_out, asset_in, amount, max_limit)
 	verify{
 		assert_eq!(T::MultiCurrency::free_balance(asset_out, &caller), 999998100000000);
-		assert_eq!(T::MultiCurrency::free_balance(asset_in, &caller), 999998772262335);
-		assert_eq!(T::MultiCurrency::free_balance(asset_in, &fee_collector), 1000000000455474);
+		assert_eq!(T::MultiCurrency::free_balance(asset_in, &caller), 999998851241411);
+	}
+
+	multi_trade_execution_sell {
+		let c in 0..1;	// if c == 1, calculate_sell is executed
+		let e in 0..1;	// if e == 1, execute_sell is executed
```

Comment:
```suggestion
		let is_execute_sell_executed in 0..1;	// if e == 1, execute_sell is executed
```

if you agree with these, you can do similar renamings in other benchmark tests